<?xml version="1.0"?>
<launch>
	<!-- CAMERA NODE (camera_umd_package) -->
	<node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
		<param name="width" type="int" value="640" />
		<param name="height" type="int" value="480" />
		<param name="fps" type="int" value="120" />
		<param name="frame" type="string" value="wide_stereo" />
		<param name="auto_focus" type="bool" value="False" />
		<param name="focus_absolute" type="int" value="0" />

		<param name="device" type="string" value="/dev/video1" />

		<param name="camera_info_url" type="string" value="file://$(find uvc_camera)/example.yaml" />
	</node>


		<!-- LABEL DETECTOR -->
		<node pkg="label_detector" type="label_detector_node" name="label_detector" output="screen" >
			<param name="input_image_topic" type="string" value="/camera/image_raw" />

			<param name="Debugging" value="true" />
			<param name="QR_Switch" value="true" />
			<param name="HZL_Switch" value="true" />
			<param name="Qrcode_side_length" value="146" />
		</node>

	<!-- RQT -->
	<node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" output="screen" />

</launch>



<!-- 	<group ns="label_detector">
	</group> -->
